app/controllers/staff/reports_controller.rb

```diff
@@ -12,15 +12,17 @@ def index
   end
 
   def show
-    @report = Report.find(id)
-    authorize @report
+    report = Report.find(id)
+    authorize report
+
+    @report_presenter = ReportPresenter.new(report)
 
     respond_to do |format|
       format.html
       format.csv do
-        fund = @report.fund
-        @projects = Activity.project.where(organisation: @report.organisation).select { |activity| activity.associated_fund == fund }
-        @third_party_projects = Activity.third_party_project.where(organisation: @report.organisation).select { |activity| activity.associated_fund == fund }
+        fund = @report_presenter.fund
```

Comment:
Since these relate to attributes on the model directly, they should probably still reference `report` rather than the presenter.